optaplanner-core/src/main/java/org/optaplanner/core/impl/solver/termination/UnimprovedTimeMillisSpentScoreDifferenceThresholdTermination.java

```diff
@@ -31,20 +32,28 @@
 
     private final long unimprovedTimeMillisSpentLimit;
     private final Score unimprovedScoreDifferenceThreshold;
+    private final Clock clock;
 
     private Queue<Pair<Long, Score>> bestScoreImprovementHistoryQueue;
     // safeTimeMillis is until when we're safe from termination
     private long solverSafeTimeMillis = -1L;
     private long phaseSafeTimeMillis = -1L;
 
     public UnimprovedTimeMillisSpentScoreDifferenceThresholdTermination(long unimprovedTimeMillisSpentLimit,
-            Score unimprovedScoreDifferenceThreshold) {
+                                                                        Score unimprovedScoreDifferenceThreshold) {
+        this(unimprovedTimeMillisSpentLimit, unimprovedScoreDifferenceThreshold, Clock.systemUTC());
+    }
+
+    protected UnimprovedTimeMillisSpentScoreDifferenceThresholdTermination(long unimprovedTimeMillisSpentLimit,
+                                                                           Score unimprovedScoreDifferenceThreshold,
+                                                                           Clock clock) {
```

Comment:
Suggestion: if you want to avoid the "align method parameters/call arguments when multiline or not" dilemma. format it like this:
```java
protected UnimprovedTimeMillisSpentScoreDifferenceThresholdTermination(
        long unimprovedTimeMillisSpentLimit,
        Score unimprovedScoreDifferenceThreshold,
        Clock clock
) {
   // body
}
```

This is compatible with both "align ... when multiline" that's part of KIE Java Conventions and Geoffrey's IDE settings that keep ignoring KIE Java Conventions in this regard :wink:.

I don't like the aligning either. I chose to follow KIE conventions so I have that rule checked but I always format like this which keeps the rule from applying.